### DISCUSSION TITLE
Out of 8 FYRTUR blinds, 1 is not reporting its position

### DISCUSSION THREAD

**Discussion by lucas-apr** [Koenkk/zigbee2mqtt#12934] [Devices]
Hello,

I have an issue with one set of Ikea FYRTUR blinds. I have 8 installed around the house and one of them is not reporting its position. I am able to discover and add to the zigbee network without any issues. I can open and close but the position is not being published via mqtt. The only time when I am able to get its position is when I click on device/Exposes/. I tried to re-pair the device multiple times with the same results. This is what being published when I click on the position refresh button:

Info 2022-06-25 08:22:36MQTT publish: topic 'zigbee2mqtt/0x680ae2fffefe6fa0', payload '{"battery":null,"linkquality":80,"position":100,"update":{"state":null},"update_available":null}'

This is what I have in the zigbee2mqtt/database.db file:

{“id”:5,“type”:“EndDevice”,“ieeeAddr”:“0x680ae2fffefe6fa0”,“nwkAddr”:52201,“manufId”:4476,“manufName”:“IKEA of Sweden”,“powerSource”:“Battery”,“modelId”:“FYRTUR block-out roller blind”,“epList”:[1],“endpoints”:{“1”:{“profId”:260,“epId”:1,“devId”:514,“inClusterList”:[0,1,3,4,5,32,258,4096,64636],“outClusterList”:[25,4096],“clusters”:{“genPollCtrl”:{“attributes”:{“checkinInterval”:172800}}},“binds”:[{“cluster”:32,“type”:“endpoint”,“deviceIeeeAddress”:“0x00124b0024c23b72”,“endpointID”:1}],“configuredReportings”:,“meta”:{}}},“appVersion”:33,“stackVersion”:99,“hwVersion”:1,“dateCode”:“20190311”,“swBuildId”:“2.3.088”,“zclVersion”:3,“interviewCompleted”:true,“meta”:{},“lastSeen”:1656157429489,“defaultSendRequestWhen”:“active”}

For comparison, this is what I have in the same database.db file next to a working blinds in the same room:

{“id”:35,“type”:“EndDevice”,“ieeeAddr”:“0x680ae2fffee9f81d”,“nwkAddr”:20142,“manufId”:4476,“manufName”:“IKEA of Sweden”,“powerSource”:“Battery”,“modelId”:“FYRTUR block-out roller blind”,“epList”:[1],“endpoints”:{“1”:{“profId”:260,“epId”:1,“devId”:514,“inClusterList”:[0,1,3,4,5,32,258,4096],“outClusterList”:[25,4096],“clusters”:{“genBasic”:{“attributes”:{“modelId”:“FYRTUR block-out roller blind”,“manufacturerName”:“IKEA of Sweden”,“powerSource”:3,“zclVersion”:3,“appVersion”:34,“stackVersion”:99,“hwVersion”:1,“dateCode”:“20190311”,“swBuildId”:“2.2.009”}},“genPollCtrl”:{“attributes”:{“checkinInterval”:172800}},“genPowerCfg”:{“attributes”:{“batteryPercentageRemaining”:17}},“closuresWindowCovering”:{“attributes”:{“currentPositionLiftPercentage”:0}}},“binds”:[{“cluster”:32,“type”:“endpoint”,“deviceIeeeAddress”:“0x00124b0024c23b72”,“endpointID”:1},{“cluster”:1,“type”:“endpoint”,“deviceIeeeAddress”:“0x00124b0024c23b72”,“endpointID”:1},{“cluster”:258,“type”:“endpoint”,“deviceIeeeAddress”:“0x00124b0024c23b72”,“endpointID”:1}],“configuredReportings”:[{“cluster”:1,“attrId”:33,“minRepIntval”:3600,“maxRepIntval”:62000,“repChange”:0},{“cluster”:258,“attrId”:8,“minRepIntval”:1,“maxRepIntval”:62000,“repChange”:1}],“meta”:{}}},“appVersion”:34,“stackVersion”:99,“hwVersion”:1,“dateCode”:“20190311”,“swBuildId”:“2.2.009”,“zclVersion”:3,“interviewCompleted”:true,“meta”:{“configured”:2},“lastSeen”:1656140704407,“defaultSendRequestWhen”:“active”}{“configured”:2},“lastSeen”:1656154877572,“defaultSendRequestWhen”:“active”}

Any assistance is appreciated.

Lucas

**Comment by davek00997744** [comment]
This is the same issue i have now, it was the latest zigbee2mqtt update...

[https://github.com/Koenkk/zigbee2mqtt/discussions/12931](url)

**Comment by lucas-apr** [comment]
ok I was able to fix my issue. I have had to stop Z2M and start it again and now everything is reporting as it should.

**Reply by davek00997744** [reply]
Thats great, i did to a stop/start but didn't make any difference, maybe i'll try another ;)

**Comment by popy2k14** [comment]
Hey guys. i had two devices where the position report was not working.
I have solved it the following:

- cover 1: Remove battery -> press down on the cover -> wait 10 seconds -> reinsert the battery
- cover 2: Z2M -> Dashboard -> cover device -> Details -> report -> pre set on the line with "currentPositionLiftPercentage" attribute

good luck

**Reply by HeinzRuetschi** [reply]
After the Z2M update to 1.42.0 yesterday, several blinds did not show the blind position any more at all.
Both solutions offered by popy2K14 march 26, 2023 have resolved my issues and with some blinds I also had to set the cover position tilt disable report as well.
One blind always showed battery at 100%. Clicking on the battery spin button in the details section resolved this as well. 
Will see if in future the value will be shown correct and get updated.
Thanks again for these valuable recommendations!

**Comment by cbarreholm** [comment]
The reporting fix solved it for me. Makes sense given the empty configuredReportings in the problem description.

**Comment by popy2k14** [comment]
Nice to hear setting the reporting fixed it for you.
